cmd/client-s3.go

```diff
@@ -36,7 +36,7 @@ import (
 
 	"github.com/minio/mc/pkg/httptracer"
 	"github.com/minio/mc/pkg/probe"
-	"github.com/minio/minio-go"
+	minio "github.com/minio/minio-go"
```

Comment:
IMO `miniogo` is more clearer than `minio`

Reply:
changed :+1: